src/data/finset/basic.lean

```diff
@@ -387,6 +398,26 @@ by rw [coe_singleton, set.singleton_subset_iff]
   {a} ⊆ s ↔ a ∈ s :=
 singleton_subset_set_iff
 
+@[simp] lemma subset_singleton_iff {s : finset α} {a : α} : s ⊆ {a} ↔ s = ∅ ∨ s = {a} :=
```

Comment:
I think the `set` version of this statement is a better simp lemma than the one you have here. Can you add
```
@[simp] lemma subset_singleton_iff {α : Type*} {s : finset α} {x : α} : s ⊆ {x} ↔ ∀ y ∈ s, y = x :=
```
and rename this one?